lms/templates/fields/field_social_link_account.underscore

```diff
@@ -0,0 +1,13 @@
+<div class="u-field-value field <%- linkClass %>">
+    <label class="u-field-title field-label" for="u-field-link-<%- id %>-default"><%- title %></label>
```

Comment:
This should not be a label because it's not a form element (and it adds a cursor hover state as a label which is confusing)